meta.yaml

```diff
@@ -21,4 +21,8 @@ spec:
         value: ''
       - name: CHE_FABRIC8_ANALYTICS_SEGMENT_WRITE_KEY
         value: ''
+      - name: CHE_FABRIC8_ANALYTICS_WOOPRA_DOMAIN_ENDPOINT
+        value: $CHE_API/fabric8-che-analytics/woopra-domain
```

Comment:
So, by default, we assume that the endpoint will be provided by the Che Server?

Reply:
That is just the endpoint currently on hosted che, but it doesn't have to be.  I can leave those blank for now.